### ISSUE TITLE
variants_filter doesn't return all variants with decreasingFactor set to 0.0

### ISSUE THREAD

**Issue by sourabhswain** [pm4py/pm4py-source#31]
The variants_filter doesn't return all the variants when the decreasingFactor is set to 0.0 . Here is the code that I wrote for the same : 

```
all_traces = variants_filter.apply_auto_filter(log, parameters={"decreasingFactor": 1.0})
all_traces = case_statistics.get_variant_statistics(all_traces)
all_traces
```

It returns all traces except the one with the lowest frequency. 

Operating System : Windows 10

**Comment by sourabhswain** [comment]
Corrected Code (decreasingFactor set to 0.0)

```
all_traces = variants_filter.apply_auto_filter(log, parameters={"decreasingFactor": 0.0})
all_traces = case_statistics.get_variant_statistics(all_traces)
all_traces
```

**Comment by Javert899** [comment]
Thanks for signaling.

Actually, there was a bug in the application of the auto filter procedure when the decreasingFactor was set to the minimum (0.0).

A fix to the bug has been proposed in the release 1.0.17 (just released on pip)
